backend/mempool-config.sample.json

```diff
@@ -33,7 +33,9 @@
     "HOST": "127.0.0.1",
     "PORT": 8332,
     "USERNAME": "mempool",
-    "PASSWORD": "mempool"
+    "PASSWORD": "mempool",
+    "COOKIE": false,
+    "COOKIE_PATH": "/path/to/bitcoin/.cookie"
```

Comment:
it would be great to have cookie work by default, but setting cookie to true here will turn it on for everybody that did not set in in his config, potentially breaking mempool if the cookie is not in the path set here, note that this is also different for different networks (mainnet cookie is in .bitcoin/.cookie, but testnet in .bitcoin/testnet3/.cookie, and not everyone has his .bitcoin in the home directory)

i think i can change the config and sample config merging to allow for some special logic to use username and password if that is configured, cookie if that is configured and the default cookie path for that network if no auth method is set in the config, or maybe even try some known .bitcoin locations to try to find it (some of these full node distros have it on /mnt for example)